docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       - "traefik.frontend.rule=Host: your.domain.tld"
 
   mongo:
-    image: mongo:3.2
+    image: mongo:3.6
```

Comment:
can you change to Mongo 4.0? see review message.

Reply:
Done. See latest commit.

Reply:
thx :hugs: